c/ladspa/Makefile

```
@@ -1,3 +1,3 @@
-default: 
-	$(CC) -Wall -Werror -O2 -c -fPIC ../ringbuf.c ../rnnoise/*.c module.c 
+default:
+	$(CC) -Wall -Werror -O2 -c -fPIC ../c-ringbuf/ringbuf.c ../rnnoise/src/*.c module.c
```

while testing this build I stumbled on the same errors that @OldiLo seemed to experience. They fixed it at https://github.com/noisetorch/NoiseTorch/pull/269/commits/cc48ea4729f725d932897009c7cc0e5a384b7855 with following patch:
```suggestion
	$(CC) -I ../rnnoise/include -Wall -Werror -O2 -c -fPIC ../c-ringbuf/ringbuf.c ../rnnoise/src/*.c module.c
```
Can you please incorporate this fix into your branch?

Also their idea to to add the `submodule update` command right into the main Makefile, seems to be a good idea - it would spare us some special step that needs to be documented and will still be regularly forgotten.

---

I've committed the suggested changes and credited @OldiLo as a co-author in the commit for them.

---

Awesome! I'll check it again later, when I'm back on my desk.